metagraph/src/cli/stats.cpp

```diff
@@ -177,7 +180,25 @@ void print_brwt_stats(const annot::binmat::BRWT& brwt) {
     }
 }
 
-void print_stats(const Annotator &annotation) {
+void print_annotation_stats(const std::string &fname, const Config &config) {
+    std::unique_ptr<annot::MultiLabelEncoded<std::string>> anno_p
+            = initialize_annotation(fname, config);
+    auto &annotation = *anno_p;
+
+    if (!annotation.load(fname)) {
+        logger->error("Cannot load annotations from file '{}'", fname);
+        exit(1);
+    }
+
+    using RowDiffCol = annot::binmat::RowDiff<annot::binmat::ColumnMajor>;
+    if (auto *rd = dynamic_cast<const RowDiffCol *>(&annotation.get_matrix())) {
+        std::string anchors_fname = utils::make_suffix(config.infbase,
+                                                       annot::binmat::kRowDiffAnchorExt);
+        if (!config.infbase.empty() && std::filesystem::exists(anchors_fname)) {
+            const_cast<RowDiffCol *>(rd)->load_anchor(anchors_fname);
```

Comment:
does it explain in the manual that one needs to keep the anchor file after constructing the annotator in order for this to work?

Reply:
This format is actually never used in practice, the manuals say you're supposed to convert it to something else and not keep the columns separately from the anchor bitmap. I guess we could remove this, could be a thing for a next refactoring.